server.js

```diff
@@ -37,7 +37,8 @@ app.use(express.static('public'))
 app.post(['/workflows/*', '/triggers/*'], (req, res) => {
     res.send('ok');
     const payload = req.body;
-    const flatPayload = flatten(payload);
+    // Slack restricts variable keys to "a combination of letters, numbers, hyphens, and underscores."
+    const flatPayload = flatten(payload, { delimiter: '_' });
```

Comment:
```suggestion
    const flatPayload = flatten(payload, { delimiter: req.query.delimiter || '.' });
```

I _believe_ this should do the trick, so long as `?delimiter=_` is added to the URL.

Reply:
@mwbrooks - unsure if you're able to accept that suggestion on behalf of @cookbr or not - if not, let me know, and I'm happy to open a new PR w/ this change for review.

Reply:
I like the approach that @amarinelli suggested here https://github.com/stevengill/pancake-studio/pull/10#discussion_r1581375329 instead. Let's use that!